### PR TITLE
21512-Remove-Multilingual-support-from-the-bootstrap-image-and-load-it-later

### PR DESCRIPTION
**Pull request by tesonep**
I removed the Multilingual packages from the bootstraped image.
Also, I removed the creation of ST files during bootstrap. They were not used anymore.
Improved the initialization of Unicode, without having the initialization in the image.
Fixed a problem with Hermes handling UndefinedVariables.

### CODE REVIEW

File: bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st [hunk 1]
```diff
@@ -6,9 +6,5 @@ prepareBootstrap
 		exportKernelPackageNames;
 		exportAllPackagesIntoMcz;
 		exportMonticelloInStFile;
-		exportMultilingualInStFile;
-		exportSUnitInStFile;
-		exportCoreTestsInStFile;
```
Comment (guillep) [inline bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st:11]: Are these not used anymore?

File: bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st [hunk 2]
```diff
@@ -6,9 +6,5 @@ prepareBootstrap
...
 		exportFileSystemInStFile;
-		exportDeprecatedInStFile;
-		exportJenkinsToolsInStFile.
```
Comment (guillep) [inline bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st:14]: Are these not used anymore?

**Review by guillep — CHANGES_REQUESTED** [review]
There are some st file generation that was removed. Is it not used anymore?

File: bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st (hunk 1 above)
Comment (tesonep) [reply bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st:11]: They are not used anymore. Now the tests of the kernel are loaded in Hermes format, so the kernel tests on the small image does not depend of Opal-Compailer. 
The Multilingual packages where loaded twice, one in the bootstrap and then with the ST. The ST was needed to perform the initialization. 
I have moved the initialization to the 03-initializeUnicode script.

File: bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st (hunk 2 above)
Comment (tesonep) [reply bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/prepareBootstrap.st:14]: The Jenkins tools are loaded by Hermes, and the deprecated file system is still used. It is in the next line of the diff.

**Review by guillep — APPROVED** [review]

**Pull request merged**